src/ray/raylet/raylet_client.cc

```diff
@@ -376,4 +376,11 @@ Status raylet::RayletClient::PinObjectIDs(const rpc::Address &caller_address,
   return grpc_client_->PinObjectIDs(request, nullptr);
 }
 
+Status raylet::RayletClient::SubscribeToPlasma(const ObjectID &object_id) {
+  flatbuffers::FlatBufferBuilder fbb;
+  auto message = protocol::CreateSubscribePlasma(fbb, to_flatbuf(fbb, object_id));
+  fbb.Finish(message);
+  return conn_->WriteMessage(MessageType::SubscribePlasma, &fbb);
```

Comment:
I'm unsure if this should be an AtomicRequestReply, as to ensure that the raylet didn't error...

Reply:
I think most of the method in this class are just using WriteMessage, this should be fine